### ISSUE TITLE
rare disease list

### ISSUE THREAD

**Issue by ValWood** [monarch-initiative/helpdesk#119]
**Please describe your question, suggestion, or concern.**

here 
https://mondo.monarchinitiative.org/
you list
Rare diseases | 10,443
Is it possible to get  this list with some query (i.e ontology subset?)

Thanks
val

If your question or suggestion is specific to Mondo, please submit it here instead: https://github.com/monarch-initiative/mondo/issues

**Comment by matentzn** [comment]
@ValWood thanks for reaching out. We have changed our strategy on the rare diseases quite a bit recently, you can read more about this here:

https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/

In essence, rather than providing a "definite RD list", we rely on RD organisations such as NORD, GARD and Orphanet to provide us with their lists of Rare Diseases. We then use these lists to compute a rare disease subset that comprises all these classes, including their grouping classes and all descendants - which is a lot more than just ~11K.  You can get a list of all terms in the Rare Disease subset with https://api.triplydb.com/s/l2UF7ZZxq - note in particular also the column with all the subsets so you get a sense of the source "inferred means" - its a child of a Rare Disease.

Now if you want to get roughly to the number of RDs that  corresponds to the (outdated) Mondo docs I would personally look at the _NORD subset_: 

https://api.triplydb.com/s/Y5MVYTv3a

Its roughly 11K, and is based on a manual review of the original list of RDs that led to the ~11K number.

In the end, we need to to remember that everyone has a different view on thousands of questions:

1. "Rare" - compared to what? World? Europe? Japan?
2. When are disease subtypes separate diseases, when are they the same and should be counted only once?
3. When is a disease a real disease, and when is it a grouping?
4. Do we include "rare" infectious diseases, or just genetic? What about rare injuries?

IMO I wouldnt get to hung up on a specific number, and I would avoid citing a "definite list".

We also publish a rare disease subset that has all that information here: http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json.

Let me know if you like to know more! Happy to chat endlessly about this!

**Comment by ValWood** [comment]
Of course, thanks for the explanation. I feel bad you needed to type all that because I only wanted to  get a list for some outreach  for "rare disease week" and I thought it would be available because I could see the count. 
I used all disease mappings instead.

**Comment by matentzn** [comment]
Oh sorry! No worries at all, I have this question often enough and didn't write it all out!

**Comment by nlharris** [comment]
Seems like this is resolved and can be closed?

**Comment by ValWood** [comment]
yes for me!
